packages/plugin/src/rules/require-description.ts

```diff
@@ -27,7 +28,7 @@ const rule: GraphQLESLintRule<[RequireDescriptionRuleConfig]> = {
     docs: {
       category: 'Schema',
       description: 'Enforce descriptions in your type definitions.',
```

Comment:
@B2o5T I think it worth mentioning that it's not only for type definitions, but also for operations (using comments syntax instead of description syntax)

Reply:
mentioned